.changes/next-release/Bug Fix-07e700f5-e536-43f7-8eee-c95d61158bf6.json

```diff
@@ -0,0 +1,4 @@
+{
+    "type": "Bug Fix",
```

Comment:
Is this really a bug-fix? I'd say it's a feature ...

Reply:
Good point - I just went off the source being an 'issue' automatically without thinking; I'll update it.